app/views/admin/workshop/_attendances.html.haml

```diff
@@ -11,7 +11,7 @@
               - else
                 %i.fa.fa-check-square-o
             - else
-              = link_to admin_workshop_invitation_path(@workshop, invitation, attending: false), class: 'cancel_attendance', title: 'Cancel RSVP', method: :put, remote: true do
+              = link_to admin_workshop_invitation_path(@workshop, invitation, attending: false), class: 'cancel_attendance', title: 'Cancel RSVP', data: { confirm: "Are you sure you want to remove #{invitation.member.full_name} from the list of attendees?" }, method: :put, remote: true do
                 %i.fa.fa-minus-square-o
```

Comment:
Shouldn't this string come from the i18n files?

Reply:
Yes if we wanted to translate this page. Currently none of the other strings on this page are coming from i18n. I guess the question is do we want to translate this page/is it priority?

Reply:
I think for consistency we should aim to have all the newly introduced text in the locale files, this should also help us translate easier moving forward. 

However I'm happy to approve the PR and we can address that seperately.

Reply:
@despo, sounds good, I'll merge this in and start a PR moving the strings into a i18n. Or we can wait to do that until the need to translate this page arises.